templates/cloud-config-cp-join.tftpl

```diff
@@ -70,6 +72,9 @@ runcmd:
   systemctl restart containerd
   systemctl enable kubelet.service
   kubeadm join --config=/root/kubeadm.join.yaml
+  %{ if CLOUD_PROVIDER == "" }
+    kubectl taint nodes --all node-role.kubernetes.io/master-
```

Comment:
This taint won't exist when `--cloud-provider` was not provided. We shouldn't need to remove this taint. 🤔